tests/wpt/web-platform-tests/html/semantics/forms/form-submission-target/form-target-iframe.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<title>Form targetted at iframe</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+async_test(function(t) {
+  window.addEventListener("load", function() {
+    var frame = document.createElement("iframe");
+    frame.name = "frame";
+    document.documentElement.appendChild(frame);
+    var form = document.createElement("form");
+    form.target = "frame";
+    form.action = "form-target-iframe-helper.py";
+    form.method = "POST";
+    var input = document.createElement("input");
+    input.name = "n";
+    form.appendChild(input);
+    document.documentElement.appendChild(form);
+    form.submit();
+    frame.addEventListener("load", function() {
+      if (frame.contentWindow.location.href != "about:blank") {
```

Comment:
Could we make this check for the actual expected value instead? Maybe using `new URL("form-target-iframe-helper.py", window.location)`?